tests/unit/test_django_utils.py

```diff
@@ -0,0 +1,18 @@
+import pathlib
+
+from procrastinate.contrib.django import utils
+
+
+def test_get_sql():
+    migration_name = "baseline-0.5.0.sql"
+
+    migration_path = (
+        (pathlib.Path(__file__).parent.parent.parent)
+        / "procrastinate"
+        / "sql"
+        / "migrations"
+    )
+    with open(migration_path / migration_name, "rb") as f:
+        expected = f.read().decode()
+
+    assert utils.get_sql(migration_name) == expected
```

Comment:
This feels like we're testing the code by reimplementing the function twice with the same code. This means we could have the same bug in both sides. We know that this file will not change. What about:

```python
def test_get_sql():
    migration_name = "baseline-0.5.0.sql"
    migration = utils.get_sql(migration_name)

	assert migration.startswith(
		"CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;"
	)
	assert len(migration.splitlines()) == 187
```

(I prefer tests to compare dynamic results to static, human-verified things)